photutils/centroids/tests/test_core.py

```diff
@@ -200,3 +220,24 @@ def test_fit2dgaussian_dof():
     data = np.ones((2, 2))
     with pytest.raises(ValueError):
         fit_2dgaussian(data)
+
+
+def test_centroid_epsf():
```

Comment:
The only `centroid_epsf` tests are for testing exceptions.  Can you add some unit tests on some simple inputs, i.e. test the `mask`, `oversampling` and `shift_val` keywords?

You mention additional `centroid_epsf` tests in #817, but I don't see any there.  In any case, the `centroid_epsf` unit tests should go in the same PR that adds the function (i.e. this PR).